certified-connectors/CobbleStone - Contract Insight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,965 @@
+            }
+          }
+        }
+      }
+    },
+    "/CSSAPI/v2/PowerAutomate/EntityListForSelect": {
+      "get": {
+        "tags": [
+          "Internal"
+        ],
+        "summary": "Retrieve List of available Entities for Select Purpose",
+        "description": "Retrieve List of available Entities for Select Purpose",
+        "operationId": "CSSAPI_EntityListSelect",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/APIEntityListResponse"
+              }
+            }
+          },
+          "400": {
+            "description": "BadRequest",
+            "schema": {
+              "$ref": "#/definitions/CSSAPICustomMessage"
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "505": {
+            "description": "HttpVersionNotSupported",
+            "schema": {
+              "$ref": "#/definitions/CSSAPICustomMessage"
+            }
+          }
+        },
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/CSSAPI/v2/PowerAutomate/EntityListForUpdate": {
+      "get": {
+        "tags": [
+          "Internal"
+        ],
+        "summary": "Retrieve List of available Entities for Update Purpose",
+        "description": "Retrieve List of available Entities for Update Purpose",
+        "operationId": "CSSAPI_EntityListUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/APIEntityListResponse"
+              }
+            }
+          },
+          "400": {
+            "description": "BadRequest",
+            "schema": {
+              "$ref": "#/definitions/CSSAPICustomMessage"
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "505": {
+            "description": "HttpVersionNotSupported",
+            "schema": {
+              "$ref": "#/definitions/CSSAPICustomMessage"
+            }
+          }
+        },
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/CSSAPI/v2/PowerAutomate/EntityListForInsert": {
+      "get": {
+        "tags": [
+          "Internal"
+        ],
+        "summary": "Retrieve List of available Entities for Insert Purpose",
+        "description": "Retrieve List of available Entities for Insert Purpose",
```

Comment:
@nizarhandal please don't copy and paste descriptions and summaries, make sure everything is clear, grammatically correct and free of typographical errors. You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).